src/main/java/ro/fortsoft/pf4j/update/UpdateManager.java

```diff
@@ -203,31 +203,6 @@ public synchronized void refresh() {
     }
 
     /**
-     * Installs a plugin given only its URL
-     * @param url the url of the plugin to install
-     * @return true if plugin was installed
-     * @deprecated - use the installPlugin(id, version) instead
-     */
-    public boolean installPlugin(String url) {
```

Comment:
To get rid of url as string. This method is no longer used by the framework itself, it now uses the `id, version` methods.  Since we're on a 0.x release anyway, there should be no reason to keep this? But I can put it back if you believe it has value.